README.md

```diff
@@ -11,11 +11,11 @@ tarball or a git checkout.
 
 pgAdmin 4 is written as a web application in Python, using jQuery and Bootstrap
 for the client side processing and UI. On the server side, Flask is being
-utilised.
```

Comment:
"utilised" is correct in English. "utilized" is the American (or Simplified English as we call it over here :-) ) spelling.

Reply:
it's my bad.